utils/node6-transform/test/test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const transformAsyncFunctions = require('../TransformAsyncFunctions');
```

Comment:
Couple of thoughts:

1. might be good to add some more test cases from other transpilers: 
closure: https://github.com/google/closure-compiler/blob/master/test/com/google/javascript/jscomp/runtime_tests/async_function_test.js
babel: https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-async-to-generator/test/fixtures 

2. I'd recommend printing the output into a golden file. I feel like for transpiling it's really nice to verify outputs are sane, and this doesn't seem like something that should churn a lot.